node/src/main/scala/com/wavesplatform/api/common/AddressTransactions.scala

```diff
@@ -72,36 +68,35 @@ object AddressTransactions {
   ): Iterable[(Height, Transaction, Boolean)] =
```

Comment:
I recommend to replace it with Iterator
```suggestion
  ): Iterator[(Height, Transaction, Boolean)] =
```